WordPress/UITestsFoundation/Globals.swift

```diff
@@ -24,6 +24,17 @@ public func navigateBack() {
     }
 }
 
+// Generic function for cases where ScreenObject times out due to long loading time
+// making the Editor Screen evaluate to 'nil'
+public func closeEditorDiscardChanges() {
```

Comment:
What do you think of the following name, to make it clearer that we are closing _and_ discarding?

```suggestion
public func closeEditorDiscardingChanges() {
```

Or, alternatively, this:

```suggestion
public func closeEditorAndDiscardChanges() {
```

(_Of the two, my favorite is the first_)